pylint/checkers/strings.py

```diff
@@ -204,6 +205,12 @@
         "Used when we detect an f-string that does not use any interpolation variables, "
         "in which case it can be either a normal string or a bug in the code.",
     ),
+    "W1310": (
+        "Using an string which should probably be a f-string",
+        "possible-f-string-as-string",
```

Comment:
```suggestion
        "f-string-syntax-in-standard-string" 
```
```suggestion
        "f-string-syntax-in-string" 
```
```suggestion
        "f-string-syntax-in-str" 
```
```suggestion
        "forgotten-f-string-prefix"
```
```suggestion
        "possible-forgotten-f-prefix"
```
Some suggestion for another name, I think the current one is a little long.